components/insight/SRC/org/openmicroscopy/shoola/env/rnd/roi/PointIterator.java

```diff
@@ -196,8 +198,9 @@ private boolean isValidPoint(int x, int y)
      * @param sizeC The number of channels.
      * @param sizeX The number of pixels along the x-axis.
      * @param sizeY The number of pixels along the y-axis.
+     * @throws ExecutionException  If {@link RawDataFacility} can't be accessed
```

Comment:
you should probably update the parameters since the ``sizeC`` etc. are no longer passed